runtime/src/claims.rs

```diff
@@ -83,35 +83,12 @@ decl_storage! {
 		Total get(total) build(|config: &GenesisConfig<T>| {
 			config.claims.iter().fold(Zero::zero(), |acc: BalanceOf<T>, &(_, n)| acc + n)
 		}): BalanceOf<T>;
+		Prefix get(prefix) config(): Vec<u8>;
```

Comment:
I had a comment here earlier that seems to have disappeared.

This should be a `Get<&'static [u8]>` param so we don't have to go to storage every time.